site/content/en/docs/administration/advanced/update_guide.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Update guide'
+linkTitle: 'Update guide'
+weight: 11
+description: 'Instructions for updating CVAT deployed with docker compose'
+---
+
+<!--lint disable heading-style-->
+
+## About CVAT data volumes
+
+Docker volumes are used to store all CVAT data:
+
+- It is highly recommended backup all CVAT data before updating, follow the
+  [backup guide](/docs/administration/advanced/backup_guide/) and backup all CVAT volumes.
+
+- Update CVAT source code by any preferable way: clone with git or download zip file from GitHub.
+  Note that you need to download the entire source code, not just the `docker-compose` configuration file.
+  - With git:
+    Run the following commands inside CVAT folder
+    ```shell
+    git checkout develop
```

Comment:
I would add instructions how to clone the repository.

Reply:
Probably we need a separate section in CVAT installation guide how to get CVAT source code. Thus here we can only provide a link.